src/sonic-build-hooks/scripts/buildinfo_base.sh

```diff
@@ -231,6 +250,7 @@ run_pip_command()
         ([ "$para" == "-c" ] || [ "$para" == "--constraint" ]) && found=y
         if [ "$para" == "install" ]; then
             install=y
+            parameters[${count}]="install --trusted-host pypi.org --trusted-host files.pythonhosted.org --index-url https://pypi.org/simple "
```

Comment:
Maybe we should not add the hard code reference to the pypi indexes, we want to make sure that the customized config work fine. Some of us have the security requirement to make sure using own pipy index, not pypi.org.

Reply:
> Maybe we should not add the hard code reference to the pypi indexes, we want to make sure that the customized config work fine. Some of us have the security requirement to make sure using own pipy index, not pypi.org.
Can we make it configurable?

Reply:
removed the hardcoded reference